docs/user-guide/kustomize.md

```diff
@@ -1,5 +1,29 @@
 # Kustomize
 
+## Declarative
+
+You can define a Kustomize application manifest in the declarative GitOps way. Here is an example:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sealed-secrets
+  namespace: argocd
+spec:
+  project: default
+  source:
+    chart: sealed-secrets
+    repoURL: https://bitnami-labs.github.io/sealed-secrets
+    path: overlays/prod
+    targetRevision: 1.16.1
+  destination:
+    server: "https://kubernetes.default.svc"
+    namespace: kubeseal
```

Comment:
This `repoURL` points to a Helm chart registry, right? It may be better to have a working example here instead. For example, managing Argo CD with Kustomize.
```suggestion
  name: argocd
  namespace: argocd
spec:
  project: default
  source:
    repoURL: https://github.com/argoproj/argo-cd
    path: manifests/cluster-install
    targetRevision: v2.5.2
  destination:
    server: "https://kubernetes.default.svc"
    namespace: argocd
```

Reply:
Thanks for the real example. The most important part of this PR is for the team to validate the statement, "If the `kustomization.yaml` file exists ... Argo CD ... using Kustomize."

Reply:
I can confirm that this is correct. Might be worth adding a link to the tool detection docs in that line.
https://argo-cd.readthedocs.io/en/stable/user-guide/tool_detection/#tool-detection

Reply:
Ah, that is 6 topics behind Kustomize and Helm. It will be more effective to merge that into "Tools" right before "Kustomize".